x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -77,7 +84,7 @@ public void testCreateAndRestoreSearchableSnapshot() throws Exception {
         final String fsRepoName = randomAlphaOfLength(10);
         final String indexName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
         final String aliasName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
-        final String restoredIndexName = randomBoolean() ? indexName : randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
+        final String restoredIndexName = true ? indexName : randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
```

Comment:
Ah sorry another lost `randomBoolean()` here, and I'm guessing that `deletedBeforeMount` doesn't make sense if restoring into a different index name.

Reply:
Thanks, I've updated and fixed the test to take all situations properly into account. I've also left the `deletedBeforeMount` + different index name situation in there (just for completeness, even if it's not the most interesting scenario to test).